config_test.go

```diff
@@ -55,7 +55,6 @@ var keysToTest = []string{
 	"PCP_PMCDOPTIONS_PATH",
 	"PCP_PMCDRCLOCAL_PATH",
 	"PCP_PMPROXYOPTIONS_PATH",
-	"PCP_PMWEBDOPTIONS_PATH",
 	"PCP_PMMGROPTIONS_PATH",
```

Comment:
The PMMGR line (58) can also be dropped nowadays.

Reply:
Done